docs/source/mturk.rst

```diff
@@ -0,0 +1,181 @@
+..
```

Comment:
Cool.

Overall, the style/formatting is different to @alexholdenmiller 's  tutorial -- what do we do about that?